internal/providers/terraform/azure/frontdoor_firewall_policy.go

```diff
@@ -0,0 +1,45 @@
+package azure
+
+import (
+	"github.com/infracost/infracost/internal/resources/azure"
+	"github.com/infracost/infracost/internal/schema"
+)
+
+// getAzureRMFrontdoorFirewallPolicyRegistryItem returns a registry item for the
+// resource
+func getAzureRMFrontdoorFirewallPolicyRegistryItem() *schema.RegistryItem {
+	return &schema.RegistryItem{
+		Name:  "azurerm_frontdoor_firewall_policy",
+		RFunc: newFrontdoorFirewallPolicy,
+		ReferenceAttributes: []string{
+			"resource_group_name",
+		},
+	}
+}
+
+// newFrontdoorFirewallPolicy parses Terraform's data and uses it to build
+// a new resource
+func newFrontdoorFirewallPolicy(d *schema.ResourceData, u *schema.UsageData) *schema.Resource {
+	region := lookupRegion(d, []string{"resource_group_name"})
+	region = regionToZone(region)
```

Comment:
Should we include check for "US Gov 1" region (exists in Pricing API), if TF doesn't list it as an available region?

Reply:
For Azure, I think TF lists the regions that are supported by the account (meaning if the user's creds supports gov cloud regions, they'll see them). Search the codebase for `US Gov` in case some of those example can be used here, e.g. `dnsQueriesCostComponent`

Reply:
Oh, good to know. I'll add the check then. Thank you!